### PR TITLE
[feat] - add bigger timeout for space06 lotus service

### PR DESCRIPTION
**Pull request by mur-me**
Super trivial change:
if variable exist -> put new annotation for timeout.
---
Refactor:
1. moved service annotations to the separate dictionary value.
2. added default value
3. rewrite value for space06 with new timeout
4. refactor Lotus and lotus variables to only `lotus`

Tested for refactoring - nothing really changed, only order:
```
uladzislau@MURAVEIKANB:~/repos/filecoin-chart$ for i in $(ls -1 values/mainnet/network/ | grep -v cache | cut -d. -f 1); do make diff NODE="$i";done
helm diff upgrade --install -f values.yaml -f values/mainnet/network/api-read-master.yaml api-read-master -n network .
network, api-read-master-lotus-service, Service (v1) has changed:
  # Source: filecoin/templates/service-lotus.yaml
    annotations:
-     prometheus.io/scrape: "true"
+     prometheus.io/path: "/debug/metrics"
      prometheus.io/port: "1234"
-     prometheus.io/path: /debug/metrics
+     prometheus.io/scrape: "true"
helm diff upgrade --install -f values.yaml -f values/mainnet/network/api-read-slave-0.yaml api-read-slave-0 -n network .
helm diff upgrade --install -f values.yaml -f values/mainnet/network/api-read-slave-1.yaml api-read-slave-1 -n network .
helm diff upgrade --install -f values.yaml -f values/mainnet/network/api-read-slave-2.yaml api-read-slave-2 -n network .
helm diff upgrade --install -f values.yaml -f values/mainnet/network/space00.yaml space00 -n network .
network, space00-lotus-service, Service (v1) has changed:
  # Source: filecoin/templates/service-lotus.yaml
    annotations:
-     prometheus.io/scrape: "true"
+     prometheus.io/path: "/debug/metrics"
      prometheus.io/port: "1234"
-     prometheus.io/path: /debug/metrics
+     prometheus.io/scrape: "true"
helm diff upgrade --install -f values.yaml -f values/mainnet/network/space06-1.yaml space06-1 -n network .
helm diff upgrade --install -f values.yaml -f values/mainnet/network/space06.yaml space06 -n network .
helm diff upgrade --install -f values.yaml -f values/mainnet/network/space07.yaml space07 -n network .
network, space07-lotus-service, Service (v1) has changed:
  # Source: filecoin/templates/service-lotus.yaml
    annotations:
-     prometheus.io/scrape: "true"
+     prometheus.io/path: "/debug/metrics"
      prometheus.io/port: "1234"
-     prometheus.io/path: /debug/metrics
+     prometheus.io/scrape: "true"
```

### CODE REVIEW

File: templates/service-lotus.yaml [hunk 1]
```diff
@@ -22,7 +20,7 @@ spec:
       port: 1234
       targetPort: 1234
       name: tcp-api
-  {{- if .Values.Lotus.service.gateway }}
+  {{- if .Values.lotus.service.gateway }}
```
Comment (witnsby) [inline templates/service-lotus.yaml:23]: Are you sure we need to have this change?

File: templates/statefulset.yaml [hunk 2]
```diff
@@ -10,8 +10,8 @@ spec:
   serviceName: {{ .Release.Name }}-lotus-service
   selector:
     matchLabels:
-      {{- if .Values.Lotus.service.is_slave }}
-      release: {{ .Values.Lotus.service.label }}
+      {{- if .Values.lotus.service.is_slave }}
```
Comment (witnsby) [inline templates/statefulset.yaml:13]: Are you sure we need to have this change?

File: templates/statefulset.yaml [hunk 3]
```diff
@@ -22,8 +22,8 @@ spec:
         chart: {{ .Chart.Version }}
         appVersion: {{ .Chart.AppVersion }}
         heritage: {{ .Release.Service }}
-        {{- if .Values.Lotus.service.is_slave }}
-        release: {{ .Values.Lotus.service.label }}
+        {{- if .Values.lotus.service.is_slave }}
```
Comment (witnsby) [inline templates/statefulset.yaml:25]: again

File: templates/statefulset.yaml [hunk 4]
```diff
@@ -115,7 +115,7 @@ spec:
           value: "true"
         - name: INFRA_LOTUS_HOME
           value: /home/lotus_user
-  {{- if .Values.Lotus.service.gateway }}
+  {{- if .Values.lotus.service.gateway }}
```
Comment (witnsby) [inline templates/statefulset.yaml:118]: Are you sure we need to have this change?

File: templates/statefulset.yaml [hunk 5]
```diff
@@ -221,7 +221,7 @@ spec:
           name: api
         - containerPort: 1235
           name: p2p
-  {{- if .Values.Lotus.service.gateway }}
+  {{- if .Values.lotus.service.gateway }}
```
Comment (witnsby) [inline templates/statefulset.yaml:224]: one more confirmation

File: values/dev/network/calibrationapi-archive.yaml [hunk 6]
```diff
@@ -6,7 +6,7 @@ nodeSelector:
 IPFS:
   enabled: false
 
-Lotus:
+lotus:
```
Comment (witnsby) [inline values/dev/network/calibrationapi-archive.yaml:9]: Are you sure we need to have this change?
I think we need to revert or make an additional changes

File: values/mainnet/network/api-read-slave-0.yaml [hunk 7]
```diff
@@ -9,7 +9,7 @@ init:
   clearRestart:
     enabled: false
 
-Lotus:
+lotus:
```
Comment (witnsby) [inline values/mainnet/network/api-read-slave-0.yaml:12]: Are you sure we need to have this change?
I think we need to revert or make additional changes

File: values/mainnet/network/api-read-slave-1.yaml [hunk 8]
```diff
@@ -12,7 +12,7 @@ init:
 image:
   repository: glif/lotus:v1.16.0
 
-Lotus:
+lotus:
```
Comment (witnsby) [inline values/mainnet/network/api-read-slave-1.yaml:15]: Are you sure we need to have this change?
I think we need to revert or make additional changes

File: values/mainnet/network/api-read-slave-2.yaml [hunk 9]
```diff
@@ -12,7 +12,7 @@ init:
 image:
   repository: glif/lotus:v1.16.0
 
-Lotus:
+lotus:
```
Comment (witnsby) [inline values/mainnet/network/api-read-slave-2.yaml:15]: Are you sure we need to have this change?
I think we need to revert or make additional changes

File: values/mainnet/network/space06-1.yaml [hunk 10]
```diff
@@ -9,7 +9,7 @@ init:
   clearRestart:
     enabled: true
 
-Lotus:
+lotus:
```
Comment (witnsby) [inline values/mainnet/network/space06-1.yaml:12]: Are you sure we need to have this change?
I think we need to revert or make additional changes

File: values/mainnet/network/space06.yaml [hunk 11]
```diff
@@ -11,10 +11,15 @@ init:
    clearRestart:
      enabled: false
 
-Lotus:
+lotus:
```
Comment (witnsby) [inline values/mainnet/network/space06.yaml:14]: Are you sure we need to have this change?
I think we need to revert or make additional changes

**Review by witnsby — CHANGES_REQUESTED** [review]

**Comment by mur-me** [comment]
Well, @witnsby thanks for review - yeah Lotus variable was in 2 part of value.yaml file.
Returned back `L`otus variable everywhere, removed small duplicated piece in values.yaml
---
Test run - nothing changed only order of annotations, new space06 annotation is already ther thus isn't shown in diff:
```
uladzislau@MURAVEIKANB:~/repos/filecoin-chart$ for i in $(ls -1 values/mainnet/network/ | grep -v cache | cut -d. -f 1); do make diff NODE="$i";done
helm diff upgrade --install -f values.yaml -f values/mainnet/network/api-read-master.yaml api-read-master -n network .
network, api-read-master-lotus-service, Service (v1) has changed:
  # Source: filecoin/templates/service-lotus.yaml
  apiVersion: v1
  kind: Service
  metadata:
    name: api-read-master-lotus-service
    labels:
      app: lotus-node-app
    annotations:
-     prometheus.io/scrape: "true"
+     prometheus.io/path: "/debug/metrics"
      prometheus.io/port: "1234"
-     prometheus.io/path: /debug/metrics
+     prometheus.io/scrape: "true"
  spec:
    type: ClusterIP
    selector:
      release: api-read-master
    ports:
      - protocol: TCP
        port: 1234
        targetPort: 1234
        name: tcp-api
helm diff upgrade --install -f values.yaml -f values/mainnet/network/api-read-slave-0.yaml api-read-slave-0 -n network .
helm diff upgrade --install -f values.yaml -f values/mainnet/network/api-read-slave-1.yaml api-read-slave-1 -n network .
helm diff upgrade --install -f values.yaml -f values/mainnet/network/api-read-slave-2.yaml api-read-slave-2 -n network .
helm diff upgrade --install -f values.yaml -f values/mainnet/network/space00.yaml space00 -n network .
network, space00-lotus-service, Service (v1) has changed:
  # Source: filecoin/templates/service-lotus.yaml
  apiVersion: v1
  kind: Service
  metadata:
    name: space00-lotus-service
    labels:
      app: lotus-node-app
    annotations:
-     prometheus.io/scrape: "true"
+     prometheus.io/path: "/debug/metrics"
      prometheus.io/port: "1234"
-     prometheus.io/path: /debug/metrics
+     prometheus.io/scrape: "true"
  spec:
    type: ClusterIP
    selector:
      release: space00
    ports:
      - protocol: TCP
        port: 1234
        targetPort: 1234
        name: tcp-api
helm diff upgrade --install -f values.yaml -f values/mainnet/network/space06-1.yaml space06-1 -n network .
helm diff upgrade --install -f values.yaml -f values/mainnet/network/space06.yaml space06 -n network .
helm diff upgrade --install -f values.yaml -f values/mainnet/network/space07.yaml space07 -n network .
network, space07-lotus-service, Service (v1) has changed:
  # Source: filecoin/templates/service-lotus.yaml
  apiVersion: v1
  kind: Service
  metadata:
    name: space07-lotus-service
    labels:
      app: lotus-node-app
    annotations:
-     prometheus.io/scrape: "true"
+     prometheus.io/path: "/debug/metrics"
      prometheus.io/port: "1234"
-     prometheus.io/path: /debug/metrics
+     prometheus.io/scrape: "true"
  spec:
    type: ClusterIP
    selector:
- 
      release: space07
    ports:
      - protocol: TCP
        port: 1234
        targetPort: 1234
        name: tcp-ap
```

File: values.yaml [hunk 12]
```diff
@@ -140,6 +135,11 @@ Lotus:
     is_slave: false
     label: some-release
     gateway: false
+    annotations:
+          prometheus.io/scrape: "true"
+          prometheus.io/port: "1234"
+          prometheus.io/path: /debug/metrics
+
```
Comment (witnsby) [inline values.yaml:142]: why did you add this annotation as default?
and why does it have so many spaces after annotations: ?

File: values/mainnet/network/space06.yaml [hunk 13]
```diff
@@ -14,6 +14,12 @@ init:
 Lotus:
   service:
     gateway: true
+    annotations:
+      # Annotation to apply timeout from https://github.com/glifio/filecoin-iac/blob/main/k8s/kong_ingress_timeout.tf
+      konghq.com/override: kong-ingress-read-timeout
+      prometheus.io/scrape: "true"
+      prometheus.io/port: "1234"
+      prometheus.io/path: /debug/metrics
```
Comment (witnsby) [inline values/mainnet/network/space06.yaml:22]: after adding this annotation block - a default annotation block will be replaced or it will be added automatically after default annotations?

**Review by witnsby — CHANGES_REQUESTED** [review]

File: values/mainnet/network/space06.yaml (hunk 13 above)
Comment (mur-me) [reply values/mainnet/network/space06.yaml:22]: It default one will be replaced by this one

File: values.yaml (hunk 12 above)
Comment (mur-me) [reply values.yaml:142]: I've added this annotation as default because it was originally part of service template file, thus I've decided that it will better to give us possibility to setup annotations in values file.

Comment (witnsby) [reply values.yaml:142]: if we have annotation which should be added everywhere.
make them to a default annotations block

if we have custom/specific annotations add them additionally to default.


we can have 2 variables one for default when we know that the default value spreads across all templates.
and the second one is custom. In case, when we want to add an extra annotation for one or a few services via overrides value files (live for space06)

File: values/mainnet/network/space06.yaml (hunk 13 above)
Comment (witnsby) [reply values/mainnet/network/space06.yaml:22]: could you look for this comment?
https://github.com/glifio/filecoin-chart/pull/68#discussion_r953934011

**Review by witnsby — APPROVED** [review]
